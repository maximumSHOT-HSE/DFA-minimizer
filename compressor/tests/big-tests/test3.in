14 84 6
1 5 0
1 11 1
1 9 2
1 14 3
1 13 4
1 2 5
2 8 0
2 14 1
2 6 2
2 5 3
2 1 4
2 2 5
3 3 0
3 3 1
3 6 2
3 4 3
3 11 4
3 3 5
4 3 0
4 9 1
4 8 2
4 3 3
4 3 4
4 9 5
5 12 0
5 7 1
5 1 2
5 2 3
5 11 4
5 13 5
6 14 0
6 12 1
6 8 2
6 9 3
6 10 4
6 5 5
7 1 0
7 8 1
7 3 2
7 3 3
7 2 4
7 8 5
8 4 0
8 1 1
8 5 2
8 3 3
8 3 4
8 10 5
9 13 0
9 9 1
9 4 2
9 8 3
9 7 4
9 10 5
10 1 0
10 8 1
10 13 2
10 6 3
10 2 4
10 3 5
11 5 0
11 8 1
11 12 2
11 9 3
11 1 4
11 13 5
12 7 0
12 7 1
12 13 2
12 12 3
12 5 4
12 2 5
13 6 0
13 2 1
13 14 2
13 11 3
13 4 4
13 4 5
14 9 0
14 6 1
14 14 2
14 13 3
14 9 4
14 8 5
11
13
13 6 13 1 12 6 12 4 9 1 8 13 12 
