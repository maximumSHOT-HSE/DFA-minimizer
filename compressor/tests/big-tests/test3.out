14 84 6
7 1 0
10 1 0
8 1 1
5 1 2
2 1 4
11 1 4
13 2 1
5 2 3
7 2 4
10 2 4
1 2 5
2 2 5
12 2 5
3 3 0
4 3 0
3 3 1
7 3 2
4 3 3
7 3 3
8 3 3
4 3 4
8 3 4
3 3 5
10 3 5
8 4 0
9 4 2
3 4 3
13 4 4
13 4 5
1 5 0
11 5 0
8 5 2
2 5 3
12 5 4
6 5 5
13 6 0
14 6 1
2 6 2
3 6 2
10 6 3
12 7 0
5 7 1
12 7 1
9 7 4
2 8 0
7 8 1
10 8 1
11 8 1
4 8 2
6 8 2
9 8 3
7 8 5
14 8 5
14 9 0
4 9 1
9 9 1
1 9 2
6 9 3
11 9 3
14 9 4
4 9 5
6 10 4
8 10 5
9 10 5
1 11 1
13 11 3
3 11 4
5 11 4
5 12 0
6 12 1
11 12 2
12 12 3
9 13 0
10 13 2
12 13 2
14 13 3
1 13 4
5 13 5
11 13 5
6 14 0
2 14 1
13 14 2
14 14 2
1 14 3
11
7
1 4 6 8 9 12 13 
